integrations/octopus/.port/spec.yaml

```diff
@@ -0,0 +1,26 @@
+description: Octopus Deploy integration for Port Ocean
+icon: Octopus
+docs: https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/octopus
+features:
+  - type: exporter
+    section: DevOps
+    resources:
+      - kind: project
+      - kind: release
+      - kind: deployment
+      - kind: target
+configurations:
+  - name: octopusApiKey
+    required: true
+    type: string
+    description: Octopus Deploy API key. For more information on creating an API key, refer to the <a href="https://octopus.com/docs/octopus-rest-api/how-to-create-an-api-key">Octopus API documentation</a>.
+    sensitive: true
+  - name: octopusUrl
+    required: true
+    type: url
+    description: The base URL of your Octopus Deploy instance. It should include the protocol (e.g., https://).

```

Comment:
can we have a default url so the user doesn't have to provide if they're using the cloud version?
also can you rename to serverUrl?